go/logger/standard.go

```diff
@@ -345,20 +356,20 @@ func (log *Standard) RemoveExternalLogger(handle uint64) {
 	defer log.externalLoggersMutex.Unlock()
 
 	delete(log.externalLoggers, handle)
+	log.externalLoggersCount--
 }
 
 func (log *Standard) logToExternalLoggers(level keybase1.LogLevel, format string, args []interface{}) {
-	log.externalLoggersMutex.RLock()
-	defer log.externalLoggersMutex.RUnlock()
-
-	// Short circuit logs that are more verbose than the current external log
-	// level.
-	if level < log.externalLogLevel {
-		return
+	e := entry{
+		level:  level,
+		format: format,
+		args:   args,
 	}
 
-	for _, externalLogger := range log.externalLoggers {
-		go externalLogger.Log(level, format, args)
+	// if buffer is full, don't block, just drop the log message
+	select {
+	case log.buffer <- &e:
+	default:
```

Comment:
Actually i like this solution better.  We want to be able to see from a log output that messages were dropped, so we don't rip out our hair trying to figure out what happened (because mesasges were silently dropped)